wetmap/src/components/newModals/tripCreatorListModal/types.ts

```diff
@@ -0,0 +1,13 @@
+export type ItineraryItem = {
+  BookingPage: string
+  created_at:  string
+  description: string
+  id:          number
+  name:        string
+  price:       string
```

Comment:
fine fore now, but might have to be changed to a number type once we actually code the trip creator, keep this in mind